### ISSUE TITLE
Bug in Validate method

### ISSUE THREAD

**Issue by nt314p** [wmcnamara/cheat-resistant-value#1]
I think you have a bug.

```cs
    private void Validate()
    {
        // This should be without the ! so we return from validation if all three match (which assumes that no correction is needed)
        if (!first.Equals(second) && second.Equals(third)) 
        {
            return;
        }
        // ...
```

**Comment by wmcnamara** [comment]
Oops youre right! I introduced that when I was switching them from ==/!= to equals so I could have it be generic for publishing.

Thanks!!

**Comment by wmcnamara** [comment]
Ill fix this later today, if you wanna submit a pr yiu can aswell

**Comment by wmcnamara** [comment]
Fixed in 9836539, thank you :D
